wolox-books/src/app/services/user.service.ts

```diff
@@ -10,20 +10,47 @@ import { environment } from 'src/environments/environment';
 })
 export class UserService {
   private readonly API_URI: string = `${environment.apiUri}`;
+  private isLoginSubject: BehaviorSubject<boolean>;
 
-  constructor(private readonly http: HttpClient) {}
+  constructor(private readonly http: HttpClient) {
+    this.isLoginSubject = new BehaviorSubject<boolean>(this.hasToken());
```

Comment:
Add this initialization in the declaration

Reply:
Done 😄